src/wazuh_modules/wm_sca.c

```diff
@@ -2040,6 +2044,13 @@ static int wm_sca_send_summary(wm_sca_t * data, int scan_id,unsigned int passed,
         cJSON_AddStringToObject(json_summary, "hash", "error_calculating_hash");
     }
 
+    if(integrity_hash_file){
+        cJSON_AddStringToObject(json_summary, "hash_file", integrity_hash_file);
+    }
+    else {
```

Comment:
Don't include the field when it wasn't able to calculate. In addition, this case is controlled when calling the `wm_sca_send_summary` only if `integrity_hash_file` exists (line 530).